### PR TITLE
Created a readme to resolve #7

### LINKED ISSUE #7: would you provide instructions?
Would you provide instructions or a link to instructions on how to run this?

### PR DESCRIPTION
**Pull request by rolfedh**

### CODE REVIEW

**Comment by rolfedh** [comment]
Please review the comments and items marked "TBD". We can remove the comments as they are resolved. I plan to verify the steps in this document and will make further clarifications afterward.

File: README.md [hunk 1]
```diff
@@ -89,13 +91,13 @@ someguy123:
 
 "I am the CEO of [@privex](https://steemit.com/@privex) - A VPS provider that accepts STEEM, is affordable, and [sells servers with a Zero Block Miss SLA](https://steemit.com/introduceyourself/@privex/we-are-privex-inc-we-accept-steem-protecting-your-privacy-in-the-cloud).
 
-I use @privex for my own witness. As a proof of their quality, I haven't missed a block since moving to them TBD, and I get almost 60 blocks per day. (As of May 2017)
+I use @privex as my own witness. As a proof of their quality, I haven't missed a block since moving to them TBD, and I get almost 60 blocks per day. (As of May 2017)
```
Comment (Someguy123) [inline README.md]: "as" is not the correct word here, "for" was correct.
"as" implies that I run witness as privex

File: README.md [hunk 2]
```diff
@@ -89,13 +91,13 @@ someguy123:
...
 
 You'll find plenty of people here on STEEM unaffiliated with us, praising our servers and network."
 
-You're free to use any other server provider. I don't recommend running a witness server on your home internet; it is unlikely to be reliable enough for that purpose.
+You're free to use any other service provider. I don't recommend running a witness server on your home internet; it is unlikely to be reliable enough for that purpose.
```
Comment (Someguy123) [inline README.md]: I believe that server provider was already correct.

Service provider is much more generic.

File: README.md [hunk 3]
```diff
@@ -89,13 +91,13 @@ someguy123:
...
 
-Be aware that certain server providers have poor networks or hardware that may cause missed blocks. So, you may need to try a few before you find one that's reliable enough for a witness node. 
+Be aware that certain service providers have poor networks or hardware that may cause missed blocks. So, you may need to try a few before you find one that's reliable enough for a witness node.
```
Comment (Someguy123) [inline README.md]: See previous comment of service providers

File: README.md [hunk 4]
```diff
@@ -176,7 +178,7 @@ This command returns the something similar to this:
 sudo ./run.sh shm_size 12G
 ```
 
-1. Open the config file in your favourite text editor. This example shows how to do it using Nano, an editor that is good for beginners.
+1. Open the config file in your favorite text editor. This example shows how to do it using Nano, an editor that is good for beginners.
```
Comment (Someguy123) [inline README.md]: "favourite" was already correct, it is the British English version, while "favorite" is the americanized version.

File: README.md [hunk 5]
```diff
@@ -247,7 +249,7 @@ You should see something similar to this:
    0.77369%   100000 of 12925066   (12282M free)
 ```
 
-If you see lots of red error messages, something went wrong. You can ask for help debugging it in the witness channel on [STEEMIT.CHAT](https://steemit.chat/channel/witness).
+If you see lots of red error messages, something went wrong. You can ask for help debugging it on the witness channel on [STEEMIT.CHAT](https://steemit.chat/channel/witness).
```
Comment (Someguy123) [inline README.md]: "in" is already correct. When referring to an online chat, "in" a chat is the best description.

**Review by Someguy123 — CHANGES_REQUESTED** [review]

**Comment by Someguy123** [comment]
Thank you for the README. Everything looks fine, however I don't agree with the various grammatical changes. See the review for more information.

File: README.md (hunk 1 above)
Comment (rolfedh) [reply README.md]: Agreed. Error on my part.

File: README.md (hunk 2 above)
Comment (rolfedh) [reply README.md]: Agreed.

File: README.md (hunk 3 above)
Comment (rolfedh) [reply README.md]: Agreed.

File: README.md (hunk 4 above)
Comment (rolfedh) [reply README.md]: Agreed.

File: README.md (hunk 5 above)
Comment (rolfedh) [reply README.md]: Agreed.

**Comment by rolfedh** [comment]
Would you also review the comments and TBD's? 
Presently, I'm trying to get the procedure to work on an AWS node with gobs of SSD disk space. (Compute optimized | c3.8xlarge | 32 vCPUs | 60 Gb memory | 2 x 320 (SSD) | - | 10 Gigabit network performance). Hopefully, that will get me past the following error: "xz: /home/ubuntu/steem-docker/data/witness_node_data_dir/blockchain/block_log: Write error: No space left on device". 
After that, I'll be making the grammatical changes and tweaking the markdown formatting.

**Pull request closed without merging**